IrisSoftware/main.py

```diff
@@ -117,14 +130,15 @@ def processing(self):
             # didBlink = self.detectBlink(eyeCoords, self.blinkDuration)
 
             # # Determine screen coordinates from eye coordinates
-            # screenX, screenY = computeScreenCoords(eyeCoords)
+            screenX, screenY = self.safeComputeCoords(eyeCoords)
+            print('moving to: ', screenX, screenY)
```

Comment:
We probabally don't want a print statement inside our main loop, since `print()` writes to an OS bound IO buffer which would slow down the main loop significantly

Reply:
Oops, forgot to remove it from testing. Thanks